### PR TITLE
doc(docker): document new BASIC Auth monitoring parameters

### PR DESCRIPTION
**Pull request by educhastenier**
Relates to [RUNTIME-543](https://bonitasoft.atlassian.net/browse/RUNTIME-543)

### CODE REVIEW

File: modules/runtime/pages/runtime-monitoring.adoc [hunk 1]
```diff
@@ -128,6 +128,14 @@ Sample extract of exposed Prometheus data:
  tomcat_servlet_request_max_seconds{name="dispatcherServlet",} 0.104
  ...
 
+This endpoint is protected with Basic access authentication to protect against DOS attacks. +
+Thus, when reaching this endpoint, you will have to manually enter the configured username and password information for your access to be granted.
+
+These credentials can be consulted and modified:
+
+* in <BONITA>/server/conf/tomcat-users.xml in a *Bonita Tomcat bundle*
```
Comment (danila-m) [inline modules/runtime/pages/runtime-monitoring.adoc]:
```suggestion
* in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
```

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -15,18 +15,25 @@ The 2022.1 is in development.
 === Prometheus monitoring active by default (Subscription editions)
 
 From Bonita 2022.1, xref:runtime-monitoring.adoc#_prometheus_publisher[the Prometheus metrics publisher] is now active by default, and can be disabled if needed. +
-Its access is now protected with Basic Auth authentication to protect against DOS attacks.
+Its access is now protected with Basic Auth authentication to protect against DOS attacks. +
+Thus, when reaching the prometheus metrics endpoint, typically https://localhost:8080/bonita/metrics, you will have to manually enter the configured username and password information for your access to be granted.
+
+These credentials can be consulted and modified:
+
+* in <BONITA>/server/conf/tomcat-users.xml in a *Bonita Tomcat bundle*
```
Comment (danila-m) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
* in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
```

**Review by danila-m — APPROVED** [review]
Two small corrections & good to go!

File: modules/runtime/pages/runtime-monitoring.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/runtime/pages/runtime-monitoring.adoc]: Ok, I'll do that

**Pull request merged**